org.eclipse.lyo.repackaging.jersey/pom.xml

```diff
@@ -0,0 +1,130 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<groupId>org.eclipse.lyo.oslc4j.core</groupId>
+	<artifactId>repackaging.jersey</artifactId>
+	<version>2.26</version>
+
+	<properties>
+		<maven.compiler.source>1.8</maven.compiler.source>
+		<maven.compiler.target>1.8</maven.compiler.target>
+		<jersey.version>${project.version}</jersey.version>

```

Comment:
2.26, if we change the project version above to the Lyo version.

Reply:
Exactly. @jadelkhoury @jamsden do you think the repackaged jersey should have a "lyo" version of 2.4.0 in the upcoming release or 2.26 of the jersey artifacts that are being repackaged here?

Reply:
no idea, but I'll try to say something smart.
This looks like a pom file with a group id "org.eclipse.lyo.oslc4j.core". so spontaneous reaction is to follow the lyo version.
But if this only about repackaging Jersey, with no Lyo-related code, then 2.26 makes sense.
There you go: No wise answer really. (as usual, a long phone chat might resolve it quicker)